src/components/Layout/Header/Header.tsx

```diff
@@ -12,6 +12,7 @@ import { Spinner } from "components/Spinner/Spinner.styled"
 
 export const Header = () => {
   const isDesktop = useMedia(theme.viewport.gte.sm)
```

Comment:
nestacilo by tady jen prepsat na toto?
```suggestion
  const isDesktop = useMedia(theme.viewport.gte.md)
```
jsem z tech media queries trochu zmateny 😄

Reply:
prave ze tam jsou 3 stavy:
1. Pro Desktop view
2. Pro Desktop view ale pro mensi obrazovky
3. Pro Mobilni view